### DISCUSSION TITLE
New Maze Pathfinding

### DISCUSSION THREAD

**Discussion by tupaschoal** [OpenRCT2/OpenRCT2#18599] [Feature requests]
There was some great work done on #10416 to implement a new maze pathfinding, but the OP didn't have the time to continue pursuing it.

Opening this discussion to track anyone else's interest in continuing the work.

**Comment by DoeEensGek** [comment]
This looks amazing. Maybe it's an idea to add this as a new mode in Ride/Attraction Operation Options. That way the original mode can be kept.
